src/main/java/ch/bfh/bti7081/presenter/NewSeminarPresenter.java

```diff
@@ -1,37 +1,38 @@
 package ch.bfh.bti7081.presenter;
 
 
+import ch.bfh.bti7081.model.ValidationConstants;
 import ch.bfh.bti7081.model.dto.SeminarDTO;
 import ch.bfh.bti7081.model.manager.SeminarCategoryManager;
 import ch.bfh.bti7081.model.manager.SeminarManager;
 import ch.bfh.bti7081.model.seminar.Seminar;
 import ch.bfh.bti7081.model.seminar.SeminarCategory;
-import ch.bfh.bti7081.model.ValidationConstants;
 import ch.bfh.bti7081.view.NewSeminarView;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
 
-import javax.annotation.PostConstruct;
 import java.time.LocalDateTime;
 import java.util.List;
 import java.util.stream.Collectors;
 
+@Component
 public class NewSeminarPresenter {
     private NewSeminarView view;
     @Autowired
     private SeminarManager seminarManager;
     @Autowired
     private SeminarCategoryManager seminarCategoryManager;
 
-    public NewSeminarPresenter(NewSeminarView view) {
+    public void setView(NewSeminarView view) {
         this.view = view;
     }
 
-    @PostConstruct
-    public void init(){
+    public List<String> getSeminarCategories() {
+        System.out.println(seminarCategoryManager);
```

Comment:
Wird nicht benötigt. Wenn etwas in die Konsole geloggt werden muss sollte ein Logger verwendet werden.

Reply:
Ups, völlig übersehen. Soooorry!